### DISCUSSION TITLE
Suspicious variable name (possible typo)

### DISCUSSION THREAD

**Discussion by shumov-ag** [eProsima/Fast-DDS#2570] [Show and tell]
Maybe "listener->" instead of "listener_->"

https://github.com/eProsima/Fast-DDS/blob/ac5f294f2b1d8897b9c3171cb7ac32220013c001/src/cpp/fastdds/publisher/DataWriterImpl.cpp#L1173-L1176

**Comment by MiguelCompany** [comment]
This was fixed in #3423
